fiboa_cli/datasets/lacuna_labels.py

```diff
@@ -0,0 +1,81 @@
+from ..convert_utils import convert as convert_
+
+
+SOURCES = "https://zenodo.org/records/11060871/files/mapped_fields_final.parquet?download=1"
+ID = "lacuna_labels"
+SHORT_NAME = "Lacuna Labels"
+TITLE = "A region-wide, multi-year set of crop field boundary labels for Africa"
+DESCRIPTION = """
+The [Lacunalabels](https://github.com/agroimpacts/lacunalabels/) repository hosts
+the analytical code and pointers to datasets
+resulting from a project to generate a continent-wide set of crop field
+labels for Africa covering the years 2017-2023. The data are intended
+for training and assessing machine learning models that can be used to
+map agricultural fields over large areas and multiple years.
+
+The project was funded by the [Lacuna Fund](https://lacunafund.org/),
+and led by [Farmerline](https://farmerline.co/), in collaboration with
+[Spatial Collective](https://spatialcollective.com/) and the
+[Agricultural Impacts Research Group](agroimpacts.info) at [Clark
+University](https://www.clarku.edu/departments/geography/).
+
+Please refer to the [technical
+report](docs/report/technical-report.pdf) for more details on the
+methods used to develop the dataset, an analysis of label quality, and
+usage guidelines.
+
+Data is published at https://zenodo.org/records/11060871 and can be used in accordance
+with Planet’s [participant license agreement for the NICFI
+contract](https://assets.planet.com/docs/Planet_ParticipantLicenseAgreement_NICFI.pdf).
+"""
+PROVIDERS = [
+    {
+        "name": "Agricultural Impacts Research Group",
+        "url": "https://github.com/agroimpacts/lacunalabels/",
+        "roles": ["producer"]
+    },
+    {
+        "name": "Planet Labs Inc.",
```

Comment:
Inc is outdated, in recent contracts I always had to use: Planet Labs PBC

```suggestion
        "name": "Planet Labs PBC",
```